packages/theme-language-server-common/src/server/startServer.ts

```diff
@@ -123,7 +123,7 @@ export function startServer(
         x
           .replace(/\\/g, '/')
           .replace(/^snippets\//, '')
-          .replace(/\..*$/, ''),
+          .replace(/\.liquid$/, ''),
```

Comment:
This isn't under any tests right now. Because it's inside `startServer` it's a little hard to pull out and test like a unit. Going to spend a bit of time exploring pulling this logic out but didn't want to block opening a PR to fix the bug.

Reply:
Oops 🙈 !

I mean, I'm not 100% sure a test would bring value / protect against refactors here. Plus I feel like extracting this combined dependency injection would just make the code harder to understand. Moving it out of scope would require you to do something like this and blerh 😅 

```ts
// You'd be forced to do something like this
const makeGetSnippetNamesForURI : ... = (filesForURI: Dependencies['fileForURI']) => async(uri: string) => {
  // ...
}

// I feel like that's where the value gained is smaller than the readability cost. 
// IMO it makes the thing harder to use / read at first glance.
// Am I crazy? WDYT?
```